CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/util/ZipUtils.java

```diff
@@ -38,4 +45,89 @@ public static void unzip(InputStream in, File destination) throws IOException {
         zis.close();
         in.close();
     }
+
+    public boolean deleteRecursive(File fileOrDirectory) {
+        if (fileOrDirectory.isDirectory()) {
+            for (File child : fileOrDirectory.listFiles()) { deleteRecursive(child); }
+        }
+        return fileOrDirectory.delete() || !fileOrDirectory.exists();
+    }
+
+    public void zipDirectory(String srcDirPath, File zipFile) {
+        List<String> zipFiles = new ArrayList<>();
```

Comment:
zipFile is defined as a method in this class, and also a variable in this function. very confusing while reading the code. could you name them differently?

Reply:
Done